### ISSUE TITLE
Spark 2.0 Pyspark Image

### ISSUE THREAD

**Issue by mariusvniekerk** [jupyter/docker-stacks#262]
For latest versions of Apache Toree (incubating) we need a docker-stacks build that has Spark 2.0 in it.
Development is going on here https://github.com/apache/incubator-toree/pull/56

```bash
RUN export APACHE_SPARK_VERSION=2.0.0 \
   wget http://apache.claz.org/spark/spark-$(APACHE_SPARK_VERSION)/spark-$(APACHE_SPARK_VERSION)-bin-hadoop2.6.tgz && \
   tar xzf spark-$(APACHE_SPARK_VERSION)-bin-hadoop2.6.tgz -C /usr/local && \
   cd /usr/local && \
   ln -s spark-$(APACHE_SPARK_VERSION)-bin-hadoop2.6 spark
```

Due to Toree using a more recent version of Akka we also require JDK8.

Pinging @parente @lbustelo

**Comment by parente** [comment]
I can set this up in a PR that goes green on travis to help @lbustelo with finishing the Toree 2.0 work. Once there's a stable Toree release that is Spark 2.0 compatible, then we can merge to master and build the official image. In the meantime, @lbustelo, you'll need to build the Dockerfile in my PR locally.

**Comment by lbustelo** [comment]
Easy enough. Thanks

Gino B.

**Comment by cgarciae** [comment]
+1 Been looking for a way to run a Jupyter node in a Spark 2.0 cluster, a naive install of Jupyter on top of the same Spark 2.0 image I am using is failing.

**Comment by mjg232** [comment]
+1 on spark 2.0 as well.  I'm stuck on #277 which seems to be fixed with spark 2.0

**Comment by parente** [comment]
We'll continue to track in #288.

**Comment by parente** [comment]
Merged #263 with Spark 2.0.2 in it. Appears in docker image tag `2410ad57203a` (currently latest).
